### PR TITLE
Admin Analytics - Zip Code Stats

### PR DESCRIPTION
**Pull request by weronica**
+ Added several ways by which admins can filter zip code search information, including but not limited to  "today," "last 7 days," "this month," and "custom date range." 
+ Modified `ZipSearch` model to include date information. 
+ Added sample search data to `populate.py`. 
+ Added helper JS functions to `custom.js` to test for valid dates (mm/dd/yyyy) and valid date ranges.
+ Some spacing fixes.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -312,5 +312,15 @@ class HTML(db.Model):
 	value = db.Column(db.Text)
 
 class ZipSearch(db.Model):
-	zip_code = db.Column(db.String(5), primary_key = True)
+	id = db.Column(db.Integer, primary_key = True)
```
Comment (sandlerben) [inline app/models.py]: Hmm, why did you change this? I thought using the zip as the primary key made a lot of sense.

File: app/templates/charts.html [hunk 2]
```diff
@@ -2,162 +2,273 @@
...
+                <option value="last-month">Last Month</option>
+                <option disabled>────────────────────</option>
+                <option value="last-7-days">Last 7 Days</option>
+                <option value="last-30-days">Last 30 Days</option>
+                <option value="last-60-days">Last 60 Days</option>
+                <option value="last-90-days">Last 90 Days</option>
+                <option value="last-12-months">Last 12 Months</option>
+                <option disabled>────────────────────</option>
+                <option value="custom-date-range">Custom Date Range</option>
+            </select>
```
Comment (sandlerben) [inline app/templates/charts.html]: Excited to try this out!!!

File: app/models.py (hunk 1 above)
Comment (weronica) [inline app/models.py]: Yes, I agree that it was a great idea prior to the addition of the date column; however, with the addition of the date column, there will be multiple instances of (e.g.) '19104' in the database, as each will have a different corresponding date. How does that sound?

Comment (sandlerben) [inline app/models.py]: True, I didn't think about that.
